spyder_kernels/console/kernel.py

```diff
@@ -274,8 +276,10 @@ def get_current_frames(self, ignore_internal_threads=True):
                     thread_name = thread_names[thread_id]
                 else:
                     thread_name = str(thread_id)
-                frames[thread_name] = stack
-        return frames
+                # Transform stack in a dict because FrameSummary objects
+                # are not compatible between versions of python
+                stack_dict[thread_name] = stacksummary_to_json(stack)
```

Comment:
```suggestion

                # Transform stack in a dict because FrameSummary objects
                # are not compatible between versions of Python
                stack_dict[thread_name] = stacksummary_to_json(stack)

```